content/docs/command-reference/metrics/diff.md

```diff
@@ -105,7 +108,7 @@ had in the previous commit:
 
 ```
 $ dvc metrics diff
-    Path       Metric   Value   Change
-metrics.json   TP       531     4
-metrics.json   AUC      0.967   0.003
+Path          Metric    Value    Change
+metrics.json  TP        531      4
+metrics.json  AUC       0.967    0.003
```

Comment:
The heading formats got affected by the change. Previously it was centered, it's left aligned now.